Clarify timer starts and stops for Blindfolded Solving (#1078)
thewca/wca-regulations: wca-guidelines.md

- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
- B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.
- B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).

> - B2f+) [CLARIFICATION] Since the competitor starts the solve by lifting the cover, penalties for starting the timer and for touching the cover while starting the attempt are omitted (see [Regulation B2b](regulations:regulation:B2b) and [Regulation B2c](regulations:regulation:B2c)).

Reviewer: maybe "omitted" could be changed to "do not apply"? I feel like that should be more understandable to our broad competitor base :)

Reviewer 2: ```suggestion
- B2f+) [CLARIFICATION] Since the competitor starts the solve by lifting the cover, penalties for starting the timer and for touching the cover while starting the attempt do not apply (see [Regulation B2b](regulations:regulation:B2b) and [Regulation B2c](regulations:regulation:B2c)).
```

Author: We already use `omitted` in the redulations BTW (for example, in [12a2](https://www.worldcubeassociation.org/regulations/#12a2) and [A7b1](https://www.worldcubeassociation.org/regulations/#A7b1)), but maybe with a slightly different meaning.

Reviewer: In both of those examples "omitted" is used in a sense of leaving something out or excluding it. When I first looked at this I was just aiming for it to be more understandable to a general audience but now that I'm looking at this again I'm not even sure if this would be a correct use of omitted. 

Swapping omitted out for its synonyms doesn't sound natural in English (the penalties are left out, the penalties are excluded) the synonyms give me a weird sense that the penalties should be, but are not in practice, applied. As a result, I believe that "do not apply" would be a better fit than omitted :)

Reviewer: Both are currently used in the sense of something not being applied or being excluded which fits their context pretty well. After reading both of those regulations, `omitted` actually seems a bit odd to use in this regulation. 

Originally I was thinking that `omitted` may just be harder to understand for competitors with a more limited English vocabulary, but after testing it out by substituting synonyms I think it may not even be a normal (maybe even correct?) usage of the word. 

`penalties... are left out` and `penalties... are excluded` both sound a bit odd in this context, giving me a feeling that the penalties should apply, but are not applied in practice. I now believe that `do not apply` would be a better fit as it sounds a bit more confident and natural (and maybe has a more correct meaning?)
